src/sentry/issues/escalating_issues_alg.py

```diff
@@ -22,7 +22,9 @@ class GroupCount(TypedDict):
     data: List[int]
 
 
-def generate_issue_forecast(data: GroupCount, start_time: datetime) -> List[IssueForecast]:
+def generate_issue_forecast(
+    data: GroupCount, start_time: datetime, version: str = "A"
```

Comment:
can we change the `version` arg to a `ForcaseThreshold` dataclass and pass that in instead of hard-coding the various threshold params in the functions?

Reply:
@armenzg @barkbarkimashark lmk if the new changes were more along the lines of what you guys were thinking